parfive/downloader.py

```diff
@@ -499,6 +539,8 @@ async def _write_worker(self, queue, file_pb, filepath):
         The downloaded chunk is put into a asyncio Queue by a download worker.
         This worker gets the chunk from the queue and write it to the file
         using the specified offset of the chunk.
+        If aiofiles is installed and PARFIVE_ENABLE_AIOFILES environment
+        variable is set, aiofiles will be used to write files instad.
```

Comment:
This isn't really correct any more, and I am not sure it's adding much in the docstring of this private function.
```suggestion
```